lib/dbservice_web/controllers/session_schedule_controller.ex

```diff
@@ -0,0 +1,131 @@
+defmodule DbserviceWeb.SessionScheduleController do
+  use DbserviceWeb, :controller
+
+  import Ecto.Query
+  alias Dbservice.Repo
+  alias Dbservice.SessionSchedules
+  alias Dbservice.Sessions.SessionSchedule
+
+  action_fallback(DbserviceWeb.FallbackController)
+
+  use PhoenixSwagger
+
+  alias DbserviceWeb.SwaggerSchema.SessionSchedule, as: SwaggerSchemaSessionSchedule
+
+  def swagger_definitions do
+    Map.merge(
+      SwaggerSchemaSessionSchedule.session_schedule(),
+      SwaggerSchemaSessionSchedule.session_schedules()
+    )
+  end
+
+  swagger_path :index do
+    get("/api/session-schedule")
+
+    parameters do
+      params(:query, :string, "The id of a session",
+        required: false,
+        name: "id"
+      )
+    end
+
+    response(200, "OK", Schema.ref(:SessionSchedules))
+  end
+
+  def index(conn, params) do
+    today_day_of_week = Calendar.Date.day_of_week_name(Date.utc_today())
+
+    query =
```

Comment:
Sorry, maybe I'm not understanding elixir code ;)

but why not filter by batch ID also here? 

The main (and only, for now) use case is to get the current session schedule for a batch, right?

Reply:
`batch_id` is being sent as a query parameter from client side. In db, let's say if we have 12 sessions (for different days and times) belonging to a same `batch_id`, then this will return only the session happening today.